src/core/globals.hpp

```diff
@@ -270,7 +271,7 @@ namespace big
 
 			NLOHMANN_DEFINE_TYPE_INTRUSIVE(self,
 				clean_player, force_wanted_level, free_cam, invisibility, local_visibility, never_wanted, no_ragdoll,
-				noclip, off_radar, super_run, no_collision, unlimited_oxygen, no_water_collision, wanted_level, god_mode,
+				noclip, off_radar, super_run, no_collision, unlimited_oxygen, no_water_collision, wanted_level, god_mode, part_water,
```

Comment:
why add it here and not at the bottom?

Reply:
Yeah I put them alphabetically out of habit but nobody else cares about this lmao